components/alert/test/alert-toast.vdiff.js

```diff
@@ -0,0 +1,36 @@
+import '../alert-toast.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const alertWithSubtextAndCloseButton = html`
+	<d2l-alert-toast no-auto-close type="critical" button-text="Do it!" open
+		subtext="I am subtext. I'm here to test the wrapping capabilities of adding subtext to these alerts, as well as other styling issues. Feel free to add to me!">
+		A message.
+	</d2l-alert-toast>
+`;
+
+describe('alert-toast', () => {
+
+	[
+		{ name: 'default', template: html`<d2l-alert-toast no-auto-close type="default" open>A default message.</d2l-alert-toast>` },
+		{ name: 'no-close', template: html`<d2l-alert-toast no-auto-close type="success" hide-close-button open>A message.</d2l-alert-toast>` },
+		{ name: 'button-close', template: html`<d2l-alert-toast no-auto-close type="warning" button-text="Do it!" open>A message.</d2l-alert-toast>` },
+		{ name: 'subtext-button-close', template: alertWithSubtextAndCloseButton }
+	].forEach(({ name, template }) => {
+		it(name, async() => {
+			const elem = await fixture(template, { viewport: { width: 700 } });
+			await expect(elem).to.be.golden({ margin: 50 });
```

Comment:
These tests had a very custom `rect` set ([see here](https://github.com/BrightspaceUI/core/blob/4b10bc2a21ce56f6cb6297607269172ce5031ac9/components/alert/test/alert-toast-helper.js#L6-L11)). We can set this `margin` to get it as close as possible to the old one (with more top margin, but still showing page position to the side/bottom).  Or we can just remove it and let the full page tests below handle testing position.
Removing `{ margin: 50 }`:
![image](https://github.com/BrightspaceUI/core/assets/13419300/234016a1-54d2-491d-b5fe-0016c3b967ed)
With `{ margin: 50 }`:
![image](https://github.com/BrightspaceUI/core/assets/13419300/61870637-d712-4c11-82fb-f293bcb40b9a)
Old golden:
![image](https://github.com/BrightspaceUI/core/assets/13419300/ff73589e-1d8e-4bb6-8e6f-1af43e9fae85)

Reply:
This feels like the kind of thing that should just be done with a full page test?

Reply:
Yeah we can switch to that too, I'm not sure why that wasn't done originally.  I'll try to get it to line up